components/features/bug-reporting/screenshot-capture.tsx

```
@@ -1 +1,476 @@
+
+						{/* Text input modal */}
+						{showTextInput && (
+							<div className="absolute inset-0 bg-black/50 flex items-center justify-center z-50">
+								<Card className="w-80">
+									<CardContent className="p-4 space-y-4">
+										<h3 className="font-semibold">Add Text Annotation</h3>
+										<Input
+											value={textInput}
+											onChange={(e) => setTextInput(e.target.value)}
+											placeholder="Enter annotation text..."
+											autoFocus
```

Biome (2.1.2)

[error] Use explicit boolean values for boolean JSX props.

Safe fix: Add explicit `true` literal for this attribute


(lint/style/noImplicitBoolean)